app/views/case_workers/claims/_claims_list.html.haml

```diff
@@ -29,8 +29,9 @@
       - present_collection(claims).each do |claim|
         %tr
           %td
-            = link_to claim.case_number , case_workers_claim_path(claim),{:class => 'js-test-case-number-link'}
-            %div.unique-id-small= claim.unique_id
+            = link_to claim.case_number , case_workers_claim_path(claim),{:class => 'js-test-case-number-link', 'aria-label': "#{claim.state.humanize}, claim I.D. #{claim.unique_id}"}
```

Comment:
@cleargif I think the user is only interested in the case number, not the database id (we only ask them to specify that if there is an issue with the claim, so we can live without it in the case of visually impaired caseworker/external user).

Also, this field can be empty (you can save a draft claim without specifying a case number) - so we need to say something like casenumber not specified yet in this case.  JLO or I can write a presenter method to do that if you want.